src/app.tsx

```diff
@@ -108,6 +137,7 @@ export const Application = () => {
                 const files = entries.map(([name, attrs]) => ({
                     ...attrs,
                     name,
+                    category: categorize(name, attrs),
                     to: info.target(name)?.type ?? null
```

Comment:
Right, we wanna re-do this at some point, but I suppose that won't change much for the function definitiion

Reply:
We're always going to need to do some kind of "special treatment" for symlinks here, so however we handle that, the category code will be part of the same.